pkg/sql/create_external_connection.go

```diff
@@ -103,8 +103,27 @@ func (p *planner) createExternalConnection(
 
 	// Construct the ConnectionDetails for the external resource represented by
 	// the External Connection.
+
+	var SkipCheckingExternalStorageConnection bool
+	var SkipCheckingKMSConnection bool
+	if params.ExecCfg().ExternalConnectionTestingKnobs != nil {
+		SkipCheckingExternalStorageConnection = params.ExecCfg().ExternalConnectionTestingKnobs.SkipCheckingExternalStorageConnection()
```

Comment:
don't we also have to check that `SkipCheckingExternal... != nil`?

Reply:
Done